dotcom-rendering/src/components/Card/Card.tsx

```diff
@@ -138,7 +138,7 @@ const starWrapper = (cardHasImage: boolean) => css`
 	}
 `;
 
-export const StarRatingComponent = ({
+const StarRatingComponent = ({
```

Comment:
On first glance I thought this would be the component itself, but I guess this is just a wrapper as well - maybe this should be renamed? 
```suggestion
const StarRatingWrapper = ({
```